keps/sig-apps/2599-minreadyseconds-for-statefulsets/README.md

```diff
@@ -403,16 +403,28 @@ This section must be completed when targeting beta to a release.
 Try to be as paranoid as possible - e.g., what if some components will restart
 mid-rollout?
 -->
+It shouldn't impact already running workloads. This is an opt-in feature since 
+users need to explicitly set the minReadySeconds parameter in the StatefulSet spec i.e `.spec.minReadySeconds` field.
+If the feature is disabled the field is preserved. If it was already set in the persisted StatefulSet object, otherwise it is silently dropped.
 
 ###### What specific metrics should inform a rollback?
 
 <!--
 What signals should users be paying attention to when the feature is young
 that might indicate a serious problem?
 -->
+`minReadySeconds` in StatefulSet doesn't get respected and all the `Ready` pods would be shown as `Available`. 
+We consider the feature to be failing if enabling the featuregate and giving
+appropriate value to minReadySeconds doesn't cause `AvailableReplicas` field to be updated 
+only after being `Ready` till minReadySeconds. The StatefulSet controller logs information about
+the number of StatefulSets without `AvailableReplicas` growing overtime which can be used by
+cluster-admin to track th failures. We also a metric called `kube_statefulset_status_replicas_available`
+which we added recently to track the number of available replicas. The cluster-admin could use
+this metric to track the problems.
```

Comment:
How would they use this metric? What are acceptable and unacceptable values for it?